tests/sentry/utils/jwt.py

```diff
@@ -0,0 +1,147 @@
+import jwt
+import pytest
+
+from sentry.utils import jwt as jwt_utils
+
+RS256_KEY = """
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I got freaked out when I saw this lol.

Reply:
I promise this is a newly generated key just for this purpose 😄 
But yes, I'm sure the security scanners will count this as "keys leaked on github" 🤷